chall05/tdawson.rb

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env ruby
+#ruby 2.5.1p57
+
+require 'open-uri'
+require 'nokogiri'
+require 'set'
+
+$WIKI_URL = String.new("https://en.wikipedia.org/wiki/")
+$seen_pages = Set.new
```

Comment:
Why using set ? I think a ruby array is enough, no?

Reply:
I used a set because I'm storing unique values (each seen page only needs to be included once) and from my understanding searching through a set is faster than through an array.